src/backend/marsha/core/storage/storage_class.py

```diff
@@ -0,0 +1,18 @@
+"""Class storage for the marsha app."""
+from django.conf import settings
+from django.core.files import storage
+from django.utils.functional import LazyObject
+
+
+class ConfiguredVideoStorage(LazyObject):
+    """Lazy object for the video storage."""
+
+    def _setup(self):
+        """Setup the video storage."""
+        params = settings.STORAGES["videos"]
+        backend = params.get("BACKEND")
+        options = params.get("OPTIONS", {})
+        self._wrapped = storage.get_storage_class(backend)(**options)
```

Comment:
in Marsha we are using django 4.2, so you can directly use `django.core.files.storage.storages` and delegate the instanciation to django.
```suggestion
        self._wrapped = storages["videos"]
```